cylc/flow/option_parsers.py

```diff
@@ -345,6 +345,11 @@ class Options(Values):
     """
 
     def __init__(self, parser, overrides=None):
+        # TODO: call parse_args() here to add our "standard options" for
+        # integration tests. These should be added in __init__() but that
+        # requires a bit of refactoring because option clashes are handled
+        # bass-ackwards ("overrides" are added before standard options).
+        parser.parse_args()
```

Comment:
@oliver-sanders - this is a bit ugly, but best for another branch if it needs fixing.